pkg/sql/sqlbase/structured.go

```diff
@@ -3122,6 +3189,84 @@ func (desc *MutableTableDescriptor) MakeMutationComplete(m DescriptorMutation) e
 					return err
 				}
 			}
+		case *DescriptorMutation_ComputedColumnSwap:
+			args := t.ComputedColumnSwap
+
+			oldCol, err := desc.FindColumnByID(args.OldColumnId)
+			if err != nil {
+				return err
+			}
+			newCol, err := desc.FindColumnByID(args.NewColumnId)
+			if err != nil {
+				return err
+			}
+
+			newCol.ComputeExpr = nil
+
+			unresolvedName := func(name string) *tree.UnresolvedName {
+				return &tree.UnresolvedName{
+					NumParts: 1,
+					Parts:    tree.NameParts{name},
+				}
+			}
+
+			// oldCol should still have new values written to newCol in case
```

Comment:
[nit]: Rephrase this as "oldCol still needs to have values written to it in case nodes read from it with a table descriptor from before the swap"